docs/spark/tutorials/get-started.md

```diff
@@ -134,7 +146,7 @@ The `dotnet` command creates a `new` application of type `console` for you. The
 
 To use .NET for Apache Spark in an app, install the Microsoft.Spark package. In your command prompt, run the following command:
 
-`dotnet add package Microsoft.Spark --version 0.6.0`
+`dotnet add package Microsoft.Spark`
```

Comment:
Any reason to remove `--version`?
 ```suggestion
`dotnet add package Microsoft.Spark --version 0.8.0`
```

Reply:
It was removed in one of your versions, so I removed it.

Reply:
If it works the same without `--version`, I think it makes sense to remove since that version will become outdated whenever a newer package is released.

Reply:
I have tries to run it without `--version`. It doesn't work for me.